frontend/app/src/components/asset-manager/AssetForm.vue

```diff
@@ -271,6 +278,27 @@ export default class AssetForm extends Vue {
     return this.assetType === ETHEREUM_TOKEN;
   }
 
+  @Watch('address')
+  async onAddressChange() {
+    if (
+      this.dontAutoFetch ||
+      !this.address.startsWith('0x') ||
+      this.address.length < 42
+    ) {
+      this.dontAutoFetch = false;
+      return;
+    }
+
+    this.fetching = true;
+    const { decimals, name, symbol } = await this.fetchTokenDetails(
+      this.address
+    );
+    this.decimals = decimals?.toString() ?? '18';
```

Comment:
I saw @yabirgb also did it like that in the backend at first, but told him not to since this endpoint is to also check if an address is an ERC20 address. If all 3 fields come null it probably isn't (even though none of the three are required by the spec)

Are you sure having default 18 here is okay? I am actually not sure after reading the spec. I asked in a tweet. Let's see. It's an interesting edge case: https://twitter.com/LefterisJP/status/1374132480350035968

Reply:
Since 2018, we had this discussion on missing decimals at least twice before. in both previous cases we went with 18 decimals when decimals were missing. We can remove it and let the user decide for sure.

Reply:
yes please do that